google-cloud-spanner/src/test/java/com/google/cloud/spanner/IntegrationTestEnv.java

```diff
@@ -58,8 +58,15 @@ public class IntegrationTestEnv extends ExternalResource {
   private InstanceAdminClient instanceAdminClient;
   private DatabaseAdminClient databaseAdminClient;
   private boolean isOwnedInstance;
+  private boolean useNewInstance;
   private RemoteSpannerHelper testHelper;
 
+  public IntegrationTestEnv(final boolean useNewInstance) {
+    this.useNewInstance = useNewInstance;
+  }
+
+  public IntegrationTestEnv() {}
```

Comment:
Should we set a default value for useNewInstance in this constructor?
```suggestion
  public IntegrationTestEnv() {
    this.useNewInstance = false;
  }
```

Reply:
A couple of nits regarding this:
1. It is common to have constructors with less parameters before constructors with more parameters (i.e. move this one to line 63)
2. It is good practice to reuse code in constructors, so I would suggest calling `this(false)` from the parameter-less constructor.